frontend/public/models/index.ts

```diff
@@ -1165,6 +1165,19 @@ export const InfrastructureModel: K8sKind = {
   crd: true,
 };
 
+export const NetworkOperatorConfigModel: K8sKind = {
+  label: 'Network',
+  labelPlural: 'Networks',
+  apiVersion: 'v1',
+  apiGroup: 'operator.openshift.io',
```

Comment:
I found multiple ways to fetch/guess the network type, but I used this because it's the only method I saw properly documented: https://docs.openshift.com/container-platform/4.7/networking/cluster-network-operator.html#nw-operator-cr_cluster-network-operator

So I decided to use the above documentation as a _source of truth_. But I might have missed the `config.openshift.io` api group. Does it provide any benefit with respect to `operatior.openshift.io`?

Reply:
It's documented as well: https://docs.openshift.com/container-platform/4.7/rest_api/config_apis/network-config-openshift-io-v1.html
We might need to ask the SDN team to know more about the difference